tools/GSDumpGUI/Forms/frmMain.cs

```diff
@@ -151,6 +206,31 @@ private void ReloadGsdxDlls()
 
             Settings.GSDXDir = gsdxFolder.FullName;
             _internalLogger.Information("Completed GSdx Loading Procedures");
+
+            string[] paths = { "", "\\plugins", "\\dll", "\\dlls" };
```

Comment:
Does that mean that there are no custom paths anymore?

Reply:
It was like that before though: [frmMain.cs@L147](https://github.com/PCSX2/pcsx2/blob/0814690534e585bf4e74da131d439ba2e087080f/tools/GSDumpGUI/Forms/frmMain.cs#L147)
`var gsdxFolder = _folderWithFallBackFinder.GetViaPatternWithFallback(Settings.GSDXDir, "*.dll", "", "plugins", "dll", "dlls");`
It checks for the selected path, which was and still is customizable, then also searches in those other places. I suppose it was a way to have dlls and dumps listed with a default path.

Another intersting option would be to have a list of paths exposed in the UI. Extra paths could then be added or removed from it.

Reply:
Ah yeah I was writing that code. See how easy I forget stuff :D